publish

```diff
@@ -26,23 +26,18 @@ fi
 
 rm -rf $CACHE_DIR/treematcher/
 
-# In the future we want to iterate over all language configs:
-# for lang_config in $LANG_CONFIGS_DIR/*.compile.js; do
+# Build and push all parsers to S3 buckets:
+for lang_config in $LANG_CONFIGS_DIR/*.compile.js; do
+  PARSELANG="$(basename $lang_config .compile.js)"
+  printf "Building and publishing $PARSELANG parser\n"
 
-# But for now just do python3:
-lang_config="$LANG_CONFIGS_DIR/python3.js"
+  rm -rf "$CACHE_DIR/$PARSELANG/"
+  ./make --lang $PARSELANG --debug >/dev/null
```

Comment:
The script's purpose is to build the parsers and push them to S3, so it only needs to indicate whether or not that was successful. If the builds fail, or the push fails, then that output is not silenced.